<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSaveFormula</class>
 <widget class="QDialog" name="dlgSaveFormula">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lao UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Infinity Calculator &gt;&gt; Save Formula</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/resources/resources/windowicon.png</selectedon>
   </iconset>
  </property>
  <widget class="QFrame" name="frmTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>215</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tblCurrentFormulae">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>301</width>
      <height>205</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string>Formula Name</string>
     </property>
     <property name="font">
      <font>
       <family>Lao UI</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Formula</string>
     </property>
     <property name="font">
      <font>
       <family>Lao UI</family>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QFrame" name="frmLocation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>421</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblFormula">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>115</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Formula</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtFormula">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>105</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #e8e8e8;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFormulaName">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Formula Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtFormulaName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #e8e8e8;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbBrowseSaveLoc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>155</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 	background-color: #dbdbdb;
    border: 2px;
	color:#464747;

}

QPushButton:hover{background-color:#40bd9e; color:white;}</string>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtSaveLocation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #e8e8e8;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSaveLocation">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdToFile">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdToMemory">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To Memory</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbSaveFormula">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>455</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lao UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 	background-color: #dbdbdb;
    border: 2px;
	color:#464747;

}

QPushButton:hover{background-color:#40bd9e; color:white;}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>455</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lao UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 	background-color: #dbdbdb;
    border: 2px;
	color:#464747;

}

QPushButton:hover{background-color:#40bd9e; color:white;}</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
